LoopKit/QuantityFormatter.swift

```diff
@@ -156,13 +167,22 @@ public extension HKUnit {
     }
 
     var preferredFractionDigits: Int {
-        if self == HKUnit.millimolesPerLiter || self == HKUnit.millimolesPerLiter.unitDivided(by: .internationalUnit()) {
+        if self == HKUnit.millimolesPerLiter || self == HKUnit.millimolesPerLiter.unitDivided(by: .internationalUnit()) || self == HKUnit.millimolesPerLiter.unitDivided(by: .minute()) {
```

Comment:
What actual values are used for `mmol/L/min`? 

IIRC, it is `2` or `3` `mg/dL/min`. Doing the simple math conversion, this is `0.11101496` and `0.16652244` `mmol/L/min`. Does Dexcom round this to `0.1` and `0.2`?

Just wondering if `1` `preferredFractionDigits` is enough for `mmol/L/min`.

Reply:
Yes, Dexcom G6 rounds this to 0.1 or 0.2 mmol/L/min (at least at the time when it was implemented).